autoprotocol/builders.py

```diff
@@ -1563,30 +1579,70 @@ def device_mode_params(
         ValueError
             If chip_material is not None or in the allowable list of tempest chip materials
         """
-        # tempest chip specification
-        if (model != "high_volume") and (model is not None):
-            raise ValueError(f"Tempest chip model is {model}. It must be: high_volume.")
-        if (nozzle != "standard") and (nozzle is not None):
-            raise ValueError(f"Tempest nozzle is {nozzle}. It must be: standard.")
-        if (chip_material not in ["silicone", "pfe"]) and (chip_material is not None):
+        if device not in ["x_mantis", "x_tempest_chip"]:
             raise ValueError(
-                f"Tempest chip material is {chip_material}. It must be either: silicone or pfe."
+                f"Device is {device}. It must be: [x_tempest_chip, x_mantis]"
             )
-
-        device_mode_params = {}
-        if any([model, chip_material, nozzle]):
-            if model is None:
-                model = "high_volume"
-            if chip_material is None:
-                chip_material = "pfe"
-            if nozzle is None:
-                nozzle = "standard"
-            x_tempest_chip = {
-                "model": model,
-                "material": chip_material,
-                "nozzle": nozzle,
+        if device is "x_mantis":
```

Comment:
Are you comparing objects (identity) or values of objects here (equality)?

Some more insight on this: https://www.openbookproject.net/thinkcs/python/english2e/ch09.html#index-2

Reply:
yeah again, should be values of objects lol. it was the friday blues. Will update!